Source/Plugins/Core/com.equella.core/scalasrc/com/tle/web/api/search/ExportCSVHelper.scala

```
@@ -203,14 +201,6 @@ object ExportCSVHelper {
     })
   }
 
-  def checkDownloadACL(): Unit = {
```

I moved this to `SearchResource` as I remember that we want to throw these exceptions in presentation layer rather than a helper.